### PR TITLE
PSMDB-1006 Release notes 4.4.13-13

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: source/release_notes/4.4.13-13.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* `SERVER-62065 <https://jira.mongodb.org/browse/SERVER-62065>`_ - Added the repairShardedCollectionChunksHistory command to restore history fields for some chunks. This aims to fix broken snapshot reads and distributed transactions.
+* `SERVER-59754 <https://jira.mongodb.org/browse/SERVER-59754>`_ - Fixed incorrect logging of queryHash/planCacheKey for operations that share the same $lookup shape
+* `SERVER-55483 <https://jira.mongodb.org/browse/SERVER-55483>`_ - Added a new startup parameter that skips verifying the table logging settings on restarting as a replica set node from the standalone mode during the restore. This speeds up the restore process.
+  
+Find the full list of changes in the `MongoDB 4.4.13 Community Edition Release notes <https://docs.mongodb.com/manual/release-notes/4.4/#4.4.13---mar-7--2022>`_.
+
+New Features
+================================================================================
+
+* :jirabug:`PSMDB-971`: Added support for KMIP encryption
```
Comment (spron-in) [inline source/release_notes/4.4.13-13.rst]: This needs more love. 

"Add support for Keys Management Interoperability Protocol (KMIP). Now users can store encryption keys in their favorite KMIP-compatible key manager to setup encryption at rest."

We should also add KMIP support into the highlights section.

Comment (spron-in) [reply source/release_notes/4.4.13-13.rst]: Add have the link to the docs in the release notes.

**Review by igorsol — APPROVED** [review]
LGTM

**Review by igorsol — COMMENTED** [review]
This is unusual situation but we already have two more bug fixes in the v4.4 branch: PSMDB-1010 and PSMDB-957. It is possible we will include them into this release.

File: source/release_notes/4.4.13-13.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+   - Clusters running versions 4.4.5 - 4.4.7 can and should upgrade to versions 4.4.11+ or 5.0.5+. 
+   
+   Note that clusters running versions 4.4.2 - 4.4.8 are affected by the bug `WT-7995 <https://jira.mongodb.org/browse/WT-7995>`_. See `WT-7995 <https://jira.mongodb.org/browse/WT-7995>`_ for specific explanation and instructions on running the `validate <https://docs.mongodb.com/manual/reference/command/validate/>`_ command to check for data inconsistencies. These data inconsistencies can lead to data loss if not identified and repaired at this point between versions 4.4.8 and 4.4.9.
+
+   If the `validate <https://docs.mongodb.com/manual/reference/command/validate/>`_  command output reports any failures, resync the impacted node from an unaffected node.   **The validate command must be run against all collections in the database. This process can be resource intensive and can negatively impact performance.**
+
+Release Highlights
+==================
+
+Added support for Keys Management Interoperability Protocol (KMIP). Now users can store encryption keys in their favorite KMIP-compatible key manager to set up encryption at rest.
```
Comment (spron-in) [inline source/release_notes/4.4.13-13.rst]:
```suggestion
* Percona Server for MongoDB now supports Keys Management Interoperability Protocol (KMIP). Users can store encryption keys in their favorite KMIP-compatible key manager to set up encryption at rest.
```

**Review by spron-in — APPROVED** [review]

**Review by igorsol — APPROVED** [review]
Thanks!

**Review by igorsol — APPROVED** [review]

File: source/release_notes/4.4.13-13.rst (hunk 1 above)
Comment (nastena1606) [reply source/release_notes/4.4.13-13.rst]: Done

**Pull request merged**